python/test/test_meas.py

```diff
@@ -221,3 +221,21 @@ def test_alp_histogram():
     print(alp_qbl("B"))
     print(alp_qbl("C"))
     print(alp_meas.map(1))
+
+def test_randomized_response_bitvec():
+    np = pytest.importorskip('numpy')
+
+    m_rr = dp.m.make_randomized_response_bitvec(
+        dp.bitvector_domain(max_weight=4), dp.discrete_distance(), f=0.95
+    )
+
+    data = np.packbits(
+        [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0]
+    )
+    assert np.array_equal(data, np.array([0, 8, 12], dtype=np.uint8))
+
+    # roundtrip: bytes -> mech -> numpy
+    release = np.frombuffer(m_rr(data.tobytes()), dtype=np.uint8)
+
+    print(np.unpackbits(data), np.unpackbits(release))
+    assert m_rr.map(1) == 0.8006676684558611
```

Comment:
again, either having this as an expression, or having math that can be checked by hand would be good. Someone coming back to this in the future will only know that this is the value that passes the test, but it will take some time to understand where it comes from.